lib/models/blueprint.js

```diff
@@ -547,16 +547,16 @@ let Blueprint = CoreObject.extend({
     const { removeTypes } = require('remove-types');
 
     if (extension === '.gts') {
-      const { parseTemplates } = require('ember-template-tag');
+      const { Preprocessor } = require('content-tag');
+      const preprocessor = new Preprocessor();
       // Strip template tags
       const templateTagIdentifier = (index) => `template = __TEMPLATE_TAG_${index}__;`;
       const templateTagIdentifierBraces = (index) => `(template = __TEMPLATE_TAG_${index}__);`;
-      const templateTagMatches = parseTemplates(code).filter((match) => match.type === 'template-tag');
+      const templateTagMatches = preprocessor.parse(code);
       let strippedCode = code;
       for (let i = 0; i < templateTagMatches.length; i++) {
         const match = templateTagMatches[i];
-        const [matchStart, matchEnd] = match.range;
-        const templateTag = code.substring(matchStart, matchEnd);
+        const templateTag = code.substring(match.range.start, match.range.end);
```

Comment:
Note, content-tag gives byte ranges, not character ranges.
See:
- https://github.com/embroider-build/content-tag/issues/45
- https://github.com/embroider-build/content-tag/issues/39#issuecomment-1832443310


Probably fine for now since the blueprints contain no multibyte characters tha

Reply:
We should probably fix this, and ideally we should just offer char ranges from content-tag to make it easy.

Reply:
Good catch!

I've added a substring util based on byte ranges and added a test case to cover this.